src/sockets/sockets.ts

```diff
@@ -1069,6 +1069,9 @@ export const server = function (io: SocketServer): void {
 
     socket = applyApplicableRewards(socket);
 
+    // Initialise matchmaking queue joined to false
+    socket.data.joined = false;
```

Comment:
This doesn’t go on the server side representation. The bool goes on the client side.